src/palladio/SOPAssign.cpp

```diff
@@ -650,16 +650,20 @@ AnnotationParsing::RangeAnnotation getRange(const AnnotationParsing::TraitParame
 };
 
 bool tryHandleEnum(SOPAssign* node, const std::wstring attrId, const std::wstring attrName, std::wstring defaultValue,
-                   const AnnotationParsing::TraitParameterMap& traitParmMap, const std::wstring& description,
-                   std::vector<std::wstring> parentFolders) {
+                   bool isFloatEnum, const AnnotationParsing::TraitParameterMap& traitParmMap,
```

Comment:
Maybe it would be good for `isFloatEnum` to be an enum instead of a bool? This would make it more clear what  the `!isFloatEnum` case means.

Reply:
using the CGAAttributeType::FLT_ENUM/STR_ENUM now